app/controllers/concerns/v2/authentication.rb

```diff
@@ -7,6 +7,8 @@ module Authentication
 
     ALLOWED_CERT_BASED_RBAC_ACTIONS = [
       { controller: 'policies', action: 'index', parents: %i[systems] },
+      { controller: 'systems', action: 'update', parents: %i[policies] },
+      { controller: 'systems', action: 'destroy', parents: %i[policies] },
```

Comment:
Are these operations restricted only to that single, cert-authd, system?

Reply:
Yes, check the tests

Reply:
https://github.com/RedHatInsights/compliance-backend/pull/2029/files#diff-42f377e9eff99098da9940c7d84cc92a4db807694709b5a14164dd831f4cf986R517-R524

and

https://github.com/RedHatInsights/compliance-backend/pull/2029/files#diff-42f377e9eff99098da9940c7d84cc92a4db807694709b5a14164dd831f4cf986R566-R574